### PR TITLE
Implement logical types

### PR DESCRIPTION
**Pull request by ivank**
Hello @Dvorsky, 10x a lot for this awesome project!
I was evaluating using it and it turns out it does not support [Avro Logical Types](https://avro.apache.org/docs/1.8.0/spec.html#Logical+Types) Which we use heavily. 

With this PR it parses all the logical types we use successfully. I've tried to use your style to the best of my ability, hopefully I've not made the change too big. Can you take a look and see if we it works for you?

### CODE REVIEW

**Review by dvorsky — CHANGES_REQUESTED** [review]
Hi @ivank, thanks for contributing! I took a look at the PR and everything seems fine. I would just like to ask you to update readme with support for logical types and maybe an example of how new CLI usage would look like? Thanks again for this nice contribution 😄

**Comment by ivank** [comment]
Hey @Dvorsky 10x for the quick response.
Here you go I hope thats ok, sorry for missing out on the README.md, completely slipped my mind :)

File: README.md [hunk 1]
```diff
@@ -43,6 +43,25 @@ npm install @degordian/avro-to-typescript --save
...
+```
+
+You can also pass them to the compilre in your code directly:
+
+    import { Compiler } from "degordian/avro-to-typescript";
+
+    const compiler = new Compiler(outputDir, { date: 'string', 'timestamp-millis': 'string'} });
+    await compiler.compile(avro);
+
+If there are logical types in avro schema but you don't provide a mapping for them, they will default to the underlying primitive type.
```
Comment (ivank) [inline README.md]:
```suggestion
If there are logical types in the avro schema but you don't provide a mapping for them, they will default to the underlying primitive type.
```

**Review by dvorsky — APPROVED** [review]

**Comment by dvorsky** [comment]
All good! Will merge and publish on npm new version.

**Pull request merged**